app/budget/services/budgetExcelService.js

```diff
@@ -0,0 +1,456 @@
+class BudgetExcelService {
+	constructor (ApiService) { // eslint-disable-line no-unused-vars
+		
+	}
+
+	generateDownload(viewState) {
+
+		// eslint-disable-next-line no-console
+		console.dir(viewState);
+
+		var data = [];
+		var termDescriptions = {
+			'05': 'Summer Session 1',
+			'06': 'Summer Special Session',
+			'07': 'Summer Session 2',
+			'08': 'Summer Quarter',
+			'09': 'Fall Semester',
+			'10': 'Fall Quarter',
+			'01': 'Winter Quarter',
+			'02': 'Spring Semester',
+			'03': 'Spring Quarter'
+		};
+
+// SUMMARY EXCEL REPORT
+		var summary = viewState.summary;
+
+		// Header
+		var header = [" "];
+		viewState.calculatedScheduleCosts.terms.forEach(function(term) {
```

Comment:
Please refactor to follow code style conventions, our current code style has been to put a new line before for loops to separate them.